routes/licenses.js

```diff
@@ -10,8 +10,21 @@ const mockResponse = [
 routes.push({
   path: '/licenses',
   method: 'GET',
-  options: {},
+  options: {
+    description: 'Licenses Index',
+    notes: 'Returns a List of all Licenses',
+  },
   handler: async (request, h) => h.response(mockResponse),
 });
 
+routes.push({
+  path: '/license/{file}',
```

Comment:
If you want to, you could already play around with including `Joi` vaidations for the `file` param here (and respectively return 400 (I guess that's appropriate, right?)